packages/frontend-2/components/settings/workspaces/billing/PricingTable/PricingTable.vue

```diff
@@ -35,6 +40,16 @@ graphql(`
   }
 `)
 
+defineEmits<{
```

Comment:
This event bubbling up two grandparent component isn't great, but with the new pricing plan design these 3 components will become one